ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -16,78 +16,216 @@
+  public void shouldRunCsStatement() throws IOException {
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject()))
+        .andReturn(ImmutableList.of(new Pair<>("CS",
+            new CreateStream(qualifiedName, Collections.emptyList(), false, Collections.emptyMap()))));
+
+    EasyMock.expect(engine.buildMultipleQueries("CS", ksqlConfig, props))
         .andReturn(Collections.emptyList());
+
+
     EasyMock.replay(engine);
+    standaloneExecutor.start();
+    EasyMock.verify(engine);
+  }
+
+  @Test
+  public void shouldRunCtStatement() throws IOException {
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject()))
+        .andReturn(ImmutableList.of(new Pair<>("CT",
+            new CreateTable(qualifiedName, Collections.emptyList(), false, Collections.emptyMap()))));
 
-    executor.start();
+    EasyMock.expect(engine.buildMultipleQueries("CT", ksqlConfig, props))
+        .andReturn(Collections.emptyList());
 
+    EasyMock.replay(engine);
+    standaloneExecutor.start();
     EasyMock.verify(engine);
   }
 
   @Test
-  public void shouldExecutePersistentQueries() {
-    final PersistentQueryMetadata query = EasyMock.niceMock(PersistentQueryMetadata.class);
-    EasyMock.expect(engine.createQueries(anyString(), eq(ksqlConfig)))
-        .andReturn(Collections.singletonList(query));
-    query.start();
-    EasyMock.expectLastCall();
-    EasyMock.replay(query, engine);
+  public void shouldRunSetStatements() throws IOException {
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject())).andReturn(ImmutableList.of(
+        new Pair<>("SET", new SetProperty(Optional.empty(), "name", "value"))
+    ));
 
-    executor.start();
+    EasyMock.replay(engine);
+    standaloneExecutor.start();
+    EasyMock.verify(engine);
 
-    EasyMock.verify(query);
+    assertThat(standaloneExecutor.getConfigProperties().get("name"), equalTo("value"));
   }
 
+
   @Test
-  public void shouldNotExecuteNonPersistentQueries() {
-    final QueryMetadata query = EasyMock.createMock(QueryMetadata.class);
-    EasyMock.expect(engine.createQueries(anyString(), eq(ksqlConfig)))
-        .andReturn(Collections.singletonList(query));
-    EasyMock.expect(query.getStatementString()).andReturn("");
-    EasyMock.replay(query, engine);
+  public void shouldRunUnSetStatements() throws IOException {
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject())).andReturn(ImmutableList.of(
+        new Pair<>("SET", new SetProperty(Optional.empty(), "name", "value")),
+        new Pair<>("UNSET", new UnsetProperty(Optional.empty(), "name"))
+    ));
+
+
+    EasyMock.replay(engine);
+    standaloneExecutor.start();
+    EasyMock.verify(engine);
+
+    assertTrue(standaloneExecutor.getConfigProperties().isEmpty());
+  }
+
+  @Test
+  @SuppressWarnings("unchecked")
+  public void shouldRunCsasStatements() throws IOException {
+
+    final Query csas1Query = EasyMock.niceMock(Query.class);
```

Comment:
Common params should be factored out to fields., e.g. factor this out to `query` field.

Reply:
Made it a field.